terragrunt/aft/main/OIDC_roles.tf

```
@@ -0,0 +1,59 @@
+locals {
+  aft_vault_cleanup = "aft-vault-cleanup-resources"
+}
+
+module "gh_oidc_roles" {
+  source      = "github.com/cds-snc/terraform-modules//gh_oidc_role?ref=v9.0.3"
+  oidc_exists = true
+
+  roles = [
+    {
+      name      = local.aft_vault_cleanup
+      repo_name = "site-reliability-engineering"
+      claim     = "ref:refs/heads/main"
+    }
+  ]
+
+  billing_tag_value = var.billing_code
+}
+
+data "aws_iam_policy_document" "aft_vault_cleanup" {
+  statement {
+    sid    = "AllowDeleteRecoveryPoints"
+    effect = "Allow"
+    actions = [
+      "backup:DeleteRecoveryPoint",
+      "backup:ListRecoveryPointsByBackupVault"
+    ]
+    resources = [
+      "arn:aws:backup:ca-central-1:137554749751:backup-vault:aft-controltower-backup-vault",
```

Since this appears to be running in the `AFT-Management` account, could we use variables for the ARNs?
```suggestion
      "arn:aws:backup:${var.region}:${var.account_id}:backup-vault:aft-controltower-backup-vault",
```